docs/contributing/Target Framework Strategy.md

```diff
@@ -58,7 +59,7 @@ This problem primarily comes from our use of polyfill APIs. To avoid this we emp
 
 This comes up in two forms:
 
-### Pattern for types 
+### Pattern for type
```

Comment:
Either "a type" or revert 🙂

Reply:
hahah .. was trying to delee a space when VS Code was hanging. Apparently deleted too much.